src/Symfony/Component/Form/Resources/translations/validators.eu.xlf

```diff
@@ -14,6 +14,126 @@
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Balio hori ez da HTML5 kolore onargarria.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Mesedez, sartu baliozko urtebetetze-eguna.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Hautatutako aukera ez da egokia.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Bilduma ez da baliozkoa.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Mesedez, hautatu baliozko kolore bat.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Mesedez, hautatu baliozko herrialde bat.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Mesedez, hautatu baliozko txanpon bat.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Mesedez, hautatu baliozko data-tarte bat.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Mesedez, sartu baliozko data eta ordua.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Mesedez, sartu baliozko data bat.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Mesedez, hautatu baliozko fitxategi bat.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Eremu ezkutua ez da baliozkoa.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Mesedez, sartu zenbaki oso bat.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Mesedez, hautatu baliozko hizkuntza bat.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Mesedez, hautatu baliozko eskualde-konfigurazio bat.</target>
```

Comment:
"Locale" is often difficult to translate. Google Translate says that this translate to "regional configuration", is this correct?
Is this the word you would use for "locale"?

Reply:
yes, it is correct.